app-frontend/src/store/initialState.ts

```diff
@@ -3,12 +3,16 @@ import { InitialState } from '../types/initialState';
 const initialState: InitialState = {
```

Comment:
duplication of initial state.
Delete this file, and compose global initial state from reducer's small initial states